scripts/release/mule/test/tests/pre/verify_signature.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -ex
+shopt -s globstar
```

Comment:
I get this error when running on mac:
$ shopt -s globstar
-bash: shopt: globstar: invalid shell option name

Reply:
These tests run within a container running ubuntu.

Reply:
Are we exclusively running these scripts in ubuntu, or would we support mac in the future?

Reply:
Is the shopt command necessary?  If not, maybe best to remove it.

Reply:
The `shopt` ensures that globstar support is turned on because it is off by default in bash.  So, this is necessary.

These scripts are meant to be run via mule which uses the ubuntu container.  If we intend to support Darwin, we'd have to test on more than just this script.

Also, bear in mind that with the exception of this test, the intent of the package tests is to test rpm and deb on linux distros.

Reply:
The find command seems to work ok with globstar turned off.  

```
ubuntu@ip-172-31-23-44:~$ shopt
autocd         	off
cdable_vars    	off
cdspell        	off
checkhash      	off
checkjobs      	off
checkwinsize   	on
cmdhist        	on
compat31       	off
compat32       	off
compat40       	off
compat41       	off
compat42       	off
compat43       	off
complete_fullquote	on
direxpand      	off
dirspell       	off
dotglob        	off
execfail       	off
expand_aliases 	on
extdebug       	off
extglob        	on
extquote       	on
failglob       	off
force_fignore  	on
globasciiranges	off
globstar       	off
gnu_errfmt     	off
histappend     	on
histreedit     	off
histverify     	off
hostcomplete   	off
huponexit      	off
inherit_errexit	off
interactive_comments	on
lastpipe       	off
lithist        	off
login_shell    	on
mailwarn       	off
no_empty_cmd_completion	off
nocaseglob     	off
nocasematch    	off
nullglob       	off
progcomp       	on
promptvars     	on
restricted_shell	off
shift_verbose  	off
sourcepath     	on
xpg_echo       	off
ubuntu@ip-172-31-23-44:~$ find . -type f -name "*.txt" -exec file {} \;
./telemetry-aggregation/telemetry-aggregator/src/main/resources/banner.txt: ASCII text
./telemetry-api/target/classes/banner.txt: ASCII text
./telemetry-api/src/main/resources/banner.txt: ASCII text
./.local/lib/python2.7/site-packages/PyYAML-5.3.1.dist-info/LICENSE.txt: UTF-8 Unicode text
./.local/lib/python2.7/site-packages/PyYAML-5.3.1.dist-info/top_level.txt: ASCII text
```
Is it really required?

Reply:
Oh, yeah, you're right, thanks.  This doesn't need globstar.  I'll remove.

I have another version that I didn't push that uses globstar, **/*.sig.